salt/elife-dashboard/config/srv-app-dashboard-app.cfg

```diff
@@ -14,7 +14,7 @@ engine: django.db.backends.postgresql
 host = {{ pillar.elife_dashboard.db.host }}
 password = {{ pillar.elife_dashboard.db.password }}
 port = {{ pillar.elife_dashboard.db.port }}
-user =
+user = {{ pillar.elife_dashboard.db.username }}
```

Comment:
`username` and `password` are fine, but `host` and `port` need to allow RDS values to override default, like in
https://github.com/elifesciences/builder-private/blob/master/salt/elife-dashboard/config/srv-app-dashboard-prod_settings.py#L25-L26